src/api-service/__app__/onefuzzlib/tasks/config.py

```diff
@@ -361,6 +361,9 @@ def build_task_config(
             else True
         )
 
+    if TaskFeature.coverage_filter in definition.features:
+        config.coverage_filter = task_config.task.coverage_filter
```

Comment:
similar to target_exe, we should probably prepend this with `{setup_dir}`.  Note, target_exe does this with `setup/` which should get changed.

Reply:
Captured in #924, but will prepend `setup/` for now, as in `target_exe`. The `Expand` API complicates this.